converged/v4/structs.go

```diff
@@ -45,11 +43,7 @@ type Client struct {
 
 // NewClient creates a new converged client
 // It initializes the V4 client and creates the service implementations
-func NewClient(credentials prismgoclient.Credentials, opts ...types.ClientOption[v4prismGoClient.Client]) (*Client, error) {
```

Comment:
Why do we not want to build this client with the credentials?

Reply:
It's easier to take v4 SDK client from cache and add it to converged v4 client

Reply:
Yeah, but now user needs to instantiate a v4 cache and a converged cache making the experience clumsy